browser-test/src/applicant/northstar_upsell.test.ts

```diff
@@ -62,9 +74,14 @@ test.describe('Upsell tests', {tag: ['@northstar']}, () => {
 
     await validateAccessibility(page)
 
-    await test.step('Validate that user can click through without logging in', async () => {
+    await test.step('Validate that user can return to the home page without logging in', async () => {
       await applicantQuestions.clickBackToHomepageButton()
-      await expect(page.locator('[data-testId="login"]')).toBeHidden()
+      // Expect the login dialog did not appear, so the user should already see the homepage
+      await expect(page.locator('[data-testId="home"]')).toBeVisible()
```

Comment:
(blocking) There should only ever be a single `body` tag. Have your locator look for that so we don't need to add the `data-testId`.